Assets/Tests/Weaver/WeaverTestLocator.cs

```diff
@@ -0,0 +1,24 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+public class WeaverTestLocator : ScriptableObject
+{
+    static string _outputDirectory;
+    public static string OutputDirectory
+    {
+        get
+        {
+            if (string.IsNullOrEmpty(_outputDirectory))
+            {
+                ScriptableObject assemblerObj = CreateInstance<WeaverTestLocator>();
+
+                var monoScript = MonoScript.FromScriptableObject(assemblerObj);
+                string myPath = AssetDatabase.GetAssetPath(monoScript);
+                _outputDirectory = Path.GetDirectoryName(myPath);
+            }
+            return _outputDirectory;
+        }
+    }
+    public static string OutputFile { get; set; }
```

Comment:
I dont think we want a static field for this, can this variable be somewhere else?